### DISCUSSION TITLE
Seems like with latest version the [Sep 20]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#3823] [Discord Threads]
> 🙋‍♂️ **Shard**  
## Seems like with latest version the default value can no longer have ' around it. Before you hat to input it like this: 'default_value', but now that gives an error that the value is not within the defined values. But default_value without ' around it works. Though the UI still shows them as needed as can be seen here:  
![image0](https://cdn.discordapp.com/attachments/661905455894888493/1019291887913422928/unknown.png)

**Comment by o1lab** [comment]
> 🙋‍♂️ **Shard**  
#### The ' is added to the default value anyways once it is saved.  
#  
> 🙋‍♂️ **Shard**  
#### So typing this:  
![image0](https://cdn.discordapp.com/attachments/1019291888127324302/1019292139831709716/unknown.png)  
#  
> 🙋‍♂️ **Shard**  
#### And clicking save still converts it to this format:  
![image0](https://cdn.discordapp.com/attachments/1019291888127324302/1019292268236111892/unknown.png)  
#  
> 🙋‍♂️ **Shard**  
#### Which makes it so that the next time you edit that column, for example change the color of one of the options, it again fails to save.  
#  
> 💁 **o1lab**  
#### <@190550716497133568>  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1019291888127324302)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
